### PR TITLE
ISPN-15059 Support different ways to handle unclean shutdowns

### PR DESCRIPTION
**Pull request by tristantarrant**
https://issues.redhat.com/browse/ISPN-15059

### CODE REVIEW

File: core/src/main/java/org/infinispan/configuration/global/UncleanShutdownAction.java [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+ * Defines the action taken when a dangling lock file is found in the persistent global state, signifying an
+ * unclean shutdown of the node (usually because of a crash or an external termination).
+ *
+ * @since 15.0
+ */
+public enum UncleanShutdownAction {
+   /**
+    * Ignores the presence of a dangling lock file in the persistent global state.
+    */
+   NONE,
```
Comment (pruivo) [inline core/src/main/java/org/infinispan/configuration/global/UncleanShutdownAction.java:13]: isn't IGNORE a better word to describe instead of NONE?

**Review by pruivo — APPROVED** [review]
LGTM, just a suggestion :+1:

File: core/src/main/java/org/infinispan/configuration/global/UncleanShutdownAction.java (hunk 1 above)
Comment (tristantarrant) [reply core/src/main/java/org/infinispan/configuration/global/UncleanShutdownAction.java]: Yes. I've changed it

**Comment by pruivo** [comment]
@tristantarrant there are some tests that need to be updated.

**Comment by tristantarrant** [comment]
Amidst nature's hue,
Verdant whispers, life's embrace,
Green breathes in the breeze.

**Comment by ryanemerson** [comment]
Thanks @tristantarrant

**Pull request merged**